skills-exercises/intro-video-skills/README.md

```diff
@@ -0,0 +1,7 @@
+# Instructions
+
+## **Submission**
+
+### Put the URL of the loom or youtube video here.
+
+### URL: https://www.loom.com/share/7aaaab0a74af46a78a452622f778f5d6
```

Comment:
Very good video. You explained every question very well. I will only suggest you to have a white/plain background because that looks good and prevent distraction.